src/book/02-projects/01-nubots/03-build-system.mdx

```diff
@@ -158,6 +158,9 @@ It is possible to specify extra arguments to `cmake` to control the configuratio
 ./b configure -- -DBUILD_TESTS=ON
 ```
 
+Another good argument to specify is `CMAKE_BUILD_TYPE`. By default, `CMAKE_BUILD_TYPE` is set to `Debug`, while this is good while you are in active development of a module, you will get very poor (slow) runtime performance.
+To get better runtime performance change `CMAKE_BUILD_TYPE` to `Release` (use `./b configure -i` if you have already run configure) and then rebuild your code.
+
```

Comment:
Maybe also add that you can run `./b configure -- -DCMAKE_BUILD_TYPE=Release`

Reply:
Will that cause cmake to reconfigure properly? Or does that only work with a fresh configuration?